Dockerfile

```diff
@@ -15,7 +15,12 @@ WORKDIR /home/user
 # ADD github /home/user/.ssh/id_rsa
 
 RUN \
-    dnf install -y --setopt=tsflags=nodocs git python38 python3-pip gcc redhat-rpm-config python3-devel python38-devel which gcc-c++ &&\
+    dnf install -y --setopt=tsflags=nodocs redhat-rpm-config which git \
+      gcc gcc-c++ \
+      python3-pip python3-devel \
+      python38 python38-devel \
+      python39 python39-devel &&\
+
```

Comment:
Is this extra empty line intentional? Won't it break continuation lines?

```suggestion
```

Reply:
yes, kept it for ease of readability.
it doesn't break the continuation of the RUN command, already tested in local. 
and aicoe/build-check also proves that.